crypto/mod.ts

```diff
@@ -61,10 +59,14 @@ const stdCrypto = ((x) => x)({
     async digest(
       algorithm: DigestAlgorithm,
       data: BufferSource | AsyncIterable<BufferSource> | Iterable<BufferSource>,
-    ): Promise<ArrayBuffer> {
+    ): Promise<ArrayBuffer | string> {
```

Comment:
I'll take a look into it, but as I said earlier, the problem with the inverted values was because of the return being a ArrayBuffer (I think it's because when I convert the hash into the ArrayBuffer type it stores it as a little-endian value, when in fact we want the big-endian value). But I'll see what I can do and let you know.

Reply:
About this, I sent the PR into the discord server to see if anyone could help me. I have ran out of ideas to make this work as a ArrayBuffer.